src/nrniv/nrnbbcore_write/callbacks/bbcore_callbacks.cpp

```diff
@@ -0,0 +1,524 @@
+#include "bbcore_callbacks.h"
```

Comment:
nitpick : can be renamed to `corenrn_callbacks` instead of `bbcore_callbacks`. Idea is to remove `bb` from most of the implementation.